base/classes.dtx

```diff
@@ -519,6 +519,14 @@
 %</article|report|book>
 %    \end{macrocode}
 %
+% \changes{1.4o}{2022/06/23}{Set \cs{stock\dots} registers}
+% No trim marks in the standard classes, so the stock and the paper are
+% the same size.
+%    \begin{macrocode}
+\stockheight=\paperheight
```

Comment:
I think you need to initialize to 0pt not `\paperheight` otherwise you can not detect when `\stockheight` has been set , which is needed to implement

> set `\pdfpageheight`  to `\stockheight`  if set, else `\paperheight`

Reply:
Updated version only creates the names, it never sets them.